web/src/components/serviceFleetCard.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Img from 'gatsby-image';
+import { Link } from 'gatsby';
+
+export default function ServiceFleetCard({ data }) {
+    const {
+        id,
+        title,
+        description,
+        image: {
+            childImageSharp: { fluid },
+        },
+    } = data;
+
+    return (
+        <Link
+            id={id}
```

Comment:
Do we need the id attr here?

Reply:
@asasmith yes for routing

Reply:
How will the id be used for routing?

Reply:
hm I was confused when I answered this. Some sections will need an id because of the secondary nav bar. I think i thought maybe it would need it but I've removed it
@asasmith